runtime/Java/src/org/antlr/v4/runtime/BufferedTokenStream.java

```diff
@@ -468,7 +468,7 @@ public String getText(Interval interval) {
 		int stop = interval.b;
         if ( start<0 || stop<0 ) return "";
         lazyInit();
-        if ( stop>=tokens.size() ) stop = tokens.size()-1;
+        if ( stop>=size() ) stop = size()-1;
```

Comment:
:exclamation: This is currently a bound checks for direct access to `tokens`. (See previous comments for details.)